source/reference/compatibility.txt

```diff
@@ -128,6 +128,32 @@ Write Preference Behavior
 legacy :binary:`mongo` shell. To disable retryable writes, use 
 :option:`--retryWrites=false <mongosh --retryWrites>`.
 
+
+ObjectId Methods and Attributes
+-------------------------------
+
+The ObjectId() methods work differently in ``mongosh`` than in the
```

Comment:
Can we link `ObjectId() methods` to this page: https://www.mongodb.com/docs/manual/reference/method/js-constructor/ ?